packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.tsx

```diff
@@ -85,6 +93,42 @@ interface DateColumnFilterProps {
   filterByTime?: boolean;
 }
 
+const CustomTextField: React.FC<TextFieldProps> = (
+  renderProps: TextFieldProps & JSX.IntrinsicAttributes
```

Comment:
You don't need to type `renderProps` manually - you've already typed it via `React.FC<TextFieldProps>`